onnxscript/function_libs/torch_lib/registration.py

```diff
@@ -40,6 +40,13 @@ def register(self, func: Any, name: str, *, private: bool = False) -> None:
     def __getitem__(self, name):
         return self._registry[name]
 
+    def get_functions(self, name: str):
```

Comment:
```suggestion
    def get(self, name: str):
```

i would suggest a more pythonic method name, or are you thinking to include methods that get_other_things?